packages/nextjs-kit/src/components/contentWithImage.tsx

```diff
@@ -35,7 +35,12 @@ export const ContentWithImage: React.FC<ContentProps> = ({
 	contentClassName = 'ps-max-w-screen lg:ps-max-w-screen-lg md:ps-max-w-screen-md sm:ps-max-w-screen-sm ps-mx-auto',
 }: ContentProps) => {
 	const router = useRouter();
+	let ResolvedImage = Image;
```

Comment:
@mitchellmarkoff For the two instances of this could you add the following comment for future us :)

```
// TODO: Remove once https://github.com/vercel/next.js/issues/52216 is resolved.
``